LiteEditor/BuildTab.cpp

```diff
@@ -144,12 +146,27 @@ void BuildTab::OnBuildEnded(clBuildEvent& e)
 
 void BuildTab::ProcessBuffer(bool last_line)
 {
+    // Limit data process frequency.
+    if(!last_line) {
+        uint64_t time_diff_ms;
+        struct timespec cur_time;
+
+        clock_gettime(CLOCK_MONOTONIC, &cur_time);
```

Comment:
this won't compile on Windows. If you are looking into applying some throttling and looking for high resolution clock, consider using `wxStopWatch` or use `std::chrono::*`